wcfsetup/install/files/lib/system/WCFSetup.class.php

```diff
@@ -360,26 +361,20 @@ protected function showSystemRequirements() {
 		$system['uploadMaxFilesize']['value'] = min(ini_get('upload_max_filesize'), ini_get('post_max_size'));
 		$system['uploadMaxFilesize']['result'] = (intval($system['uploadMaxFilesize']['value']) > 0);
 		
-		// gdlib version
-		$system['gdLib']['value'] = '0.0.0';
-		if (function_exists('gd_info')) {
-			$temp = gd_info();
-			$match = [];
-			if (preg_match('!([0-9]+\.[0-9]+(?:\.[0-9]+)?)!', $temp['GD Version'], $match)) {
-				if (preg_match('/^[0-9]+\.[0-9]+$/', $match[1])) $match[1] .= '.0';
-				$system['gdLib']['value'] = $match[1];
-			}
+		// graphics library
+		$system['graphicsLibrary']['result'] = false;
+		$system['graphicsLibrary']['value'] = '';
+		if (ImagickImageAdapter::isSupported() && ImagickImageAdapter::supportsAnimatedGIFs(ImagickImageAdapter::getVersion())) {
```

Comment:
I believe the check for animated GIFs is incorrect here. GD does not support animated GIFs at all, thus the support within Imagick is a benefit, but not a requirement.

Reply:
This check is correct, animated GIFs in ImageMagick are supported since mid 2007 (ImageMagick 6.3.6). It's a (mostly obsolete) safeguard against bogus setups.